### ISSUE TITLE
Cannot delete last row in grid

### ISSUE THREAD

**Issue by iMuMz** [64robots/nova-fields#119]
I am unable to delete the last row from the Grid.

If I click remove (x) and update the application, the data remains and it not cleared from the grid.

Nova Version : 2.0.3
64robots Version : 0.77

![Screenshot 2019-09-02 at 11 51 56](https://user-images.githubusercontent.com/22936672/64106492-928d1580-cd78-11e9-8f36-662ceae8cbca.png)

**Comment by otrsw** [comment]
Same issue

**Comment by disitec** [comment]
Thanks for the report. We have identified the issue and we are working on a fix.

**Comment by disitec** [comment]
Even though I could identify the issue in nova 2.1 it looks like in nova 2.3.1 I can't reproduce it anymore. Can you please check upgrading to latest nova version?

**Comment by disitec** [comment]
And please, also upgrade nova-fields to 0.11.1

Thanks!

**Comment by temp1029** [comment]
I'm still seeing this issue as of 0.14.0. Any chance this could be re-opened/looked into?

**Comment by disitec** [comment]
Hi, can you share a minimal reproduction and your Nova version? Will take a look onto it.

**Comment by temp1029** [comment]
Laravel Version: 2.6.1

I thought it would be easier if I send over my code and a GIF of what I'm seeing. Sorry for the large screen size, I should've reduced it before recording.
```
Row::make('Filters', [
    R64Text::make('Replace This', 'search')->rules('required'),
    R64Text::make('With This', 'replace')->rules('required'),
], 'field_filters')->childConfig([
    'fieldClasses' => 'w-full px-8 py-6',
    'hideLabelInForms' => true,
])->hideHeadingWhenEmpty()->addRowText('Add Filter'),
```
![RowFieldNoEmpty](https://user-images.githubusercontent.com/4396942/69755724-e06eab00-1126-11ea-9982-e18b9000edf4.gif)

**Comment by disitec** [comment]
Fixed and working on `0.15.0`

**Comment by temp1029** [comment]
Hey, just an update that this is fixed for me. Sorry for the long delay in testing and thanks for the quick response!
